src/common/logger.ts

```diff
@@ -1,87 +1,55 @@
-import * as Sentry from '@sentry/node'
-import * as fs from 'fs'
-import _get from 'lodash/get'
-import * as path from 'path'
+import { AsyncLocalStorage } from 'async_hooks'
 import { createLogger, format, transports } from 'winston'
-import Transport from 'winston-transport'
 
-import { isProd, isTest } from 'common/environment'
+import { environment } from 'common/environment'
 
-const logPath = 'logs'
+export const contextStorage = new AsyncLocalStorage<Map<string, string>>()
 
-// create logs dir if it does not exist
-if (!fs.existsSync(logPath)) {
-  fs.mkdirSync(logPath)
-}
-
-/**
- * Custom winston transport for Sentry.
- *
- */
-class SentryTransport extends Transport {
-  constructor(opts?: Transport.TransportStreamOptions) {
-    super(opts)
+const setContext = format((info, _) => {
+  const context = contextStorage.getStore()
+  if (context) {
+    info.requestId = context!.get('requestId')
+    // info.jobId = context!.get('jobId')
   }
+  return info
+})
 
-  log(info: any, next: () => void) {
-    if (info.level === 'error') {
-      const code = _get(info, 'extensions.code')
-
-      switch (code) {
-        case 'CODE_EXPIRED':
-        case 'UNAUTHENTICATED':
-        case 'USER_EMAIL_NOT_FOUND':
-        case 'USER_USERNAME_EXISTS':
-        case 'USER_PASSWORD_INVALID': {
-          // Ingore errors
-          break
-        }
-        default: {
-          const sentryError = new Error(info.message)
-          sentryError.stack = info.stack
-          Sentry.captureException(sentryError)
-          break
-        }
-      }
-    }
-    next()
+const createWinstonLogger = (name: string, level: string) =>
+  createLogger({
+    level,
+    format: format.combine(
+      format.splat(),
+      format.errors({ stack: true }),
+      setContext(),
+      format.label({ label: name }),
+      format.timestamp({
+        format: 'YYYY-MM-DD HH:mm:ss',
+      }),
+      format.printf(
+        (info) =>
+          `${info.timestamp} ${info.requestId ?? '-'} ${info.label} ${
+            info.level
```

Comment:
For readability, shall we add marks around meta info? e.g. `${info.timestamp} ${info.requestId ?? '-'} (${info.label}) [${info.level}]: ...`